test/runtests.jl

```diff
@@ -458,83 +458,6 @@ end
     @test _x509_certificate.issuer_name == x509_certificate.issuer_name
 end
 
-# https://www.openssl.org/docs/man3.0/man7/OSSL_PROVIDER-legacy.html
-@testset "Encrypt" begin
-    if OpenSSL.version_number() ≥ v"3"
-        OpenSSL.load_legacy_provider()
-    end
-    evp_ciphers = [
-        EvpEncNull(),
-        EvpBlowFishCBC(), # legacy
-        EvpBlowFishECB(), # legacy
-        #EvpBlowFishCFB(), // not supported
-        EvpBlowFishOFB(), # legacy
-        EvpAES128CBC(),
```

Comment:
`EvpAES128CBC`, `EvpAES128ECB` and `EvpAES128OFB` aren't marked as legacy and should remain.